maas2netbox/validators.py

```diff
@@ -131,8 +130,8 @@ def check_nodes(self):
         for node in self.netbox_nodes:
             try:
                 if node['custom_fields']['IPMI']:
-                    declared_ipmi_address = urlparse(
-                        node['custom_fields']['IPMI']).netloc
+                    declared_ipmi_address = utils.get_hostname(
```

Comment:
Please move the method inside this class as a static method rather than creating a new file

Reply:
Added as a static method on the base class Validator.